### PR TITLE
Merge Basic Drivebase

### PR DESCRIPTION
**Pull request by ryan-huang1**
Tested on Virtual Meeting Saturday 1.22.22 without problems. Implemented suggestions from [PR 4: "Add Drivebase"](https://github.com/Team-4795/2022-Rapid-React/pull/4)

### CODE REVIEW

File: src/main/java/frc/robot/commands/arcadeDrive.java [hunk 1]
```diff
@@ -38,7 +38,7 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    //m_drivebaseSubsystem.arcadeDrive(speedValue.get() * (1 - throttleSupplier.get() * 0.3), roationValue.get() * (1 - throttleSupplier.get() * 0.3));
+    m_drivebaseSubsystem.arcadeDrive(speedValue.get() * (1 - throttleSupplier.get() * 0.3), roationValue.get() * (1 - throttleSupplier.get() * 0.3));
```
Comment (akiyeng2) [inline src/main/java/frc/robot/commands/arcadeDrive.java:41]: lol spelling

**Review by akiyeng2 — APPROVED** [review]
Looks good - only very minor things but good enough to be merged IMO

File: src/main/java/frc/robot/commands/arcadeDrive.java (hunk 1 above)
Comment (ryan-huang1) [reply src/main/java/frc/robot/commands/arcadeDrive.java:41]: bruhhh

**Review by fategg555 — APPROVED** [review]
Looks good!

**Pull request merged**